qiskit/optimization/algorithms/__init__.py

```diff
@@ -28,34 +28,43 @@
    :nosignatures:
 
    OptimizationAlgorithm
+   OptimizationResult
```

Comment:
Since this class is more than an abstract base class, but is a result object that is often returned by an Optimizer that does not sub-class it to add extra fields, it may perhaps be better in the other list directly in with Algorithms and results.

Reply:
It makes sense. I move it.